Gogacz-AllReportsView

StrongMen120 opened the pull request:
Dodanie Obsługi Przycisków

StrongMen120 committed:
Gogacz-AllReportsView

Dodanie Obsługi Przycisków

MikolajczukKamil reviewed (commented):
#113
Pięknie ładnie tylko raporty są nadal ustawiona na sztywno trzeba je pobrać z api

StrongMen120 commented:
@MikolajczukKamil Wiem dokładnie o tym tylko chciałem wysłać chociaż coś żebyście wiedzieli że będę z tym walczyć :D

StrongMen120 committed:
Gogacz Zmiana wyglądu AllReportsPage

Zmieniłem ScrollView Na ListView + dodawanie raportów z listy Lecz mam problem z wyciągnięciem Reportów Api.getReportHeaders()
